src/App.js

```diff
@@ -6,7 +6,7 @@ function App() {
     <div className="App">
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
-        <div>This is a great lesson. We are studying Git Flow</div>
```

Comment:
Looks reasonable. But maybe we should use ! instead of a dot?

Reply:
Done.